guides/cookbook.md

```diff
@@ -879,6 +1137,71 @@ be used to determine whether an Ion `int` can be fully represented as a `number`
 if not, call `Reader.bigIntValue()` to avoid loss of precision.
 </div>
 
+<div class="tabpane Python" markdown="1">
+Use of separate APIs to read different numeric types is not necessary in Ion-Python. Via both
```

Comment:
Is "Ion-Python" a spelling we use?  this is the first time i've seen it; consider rewording to avoid.

"Use of separate APIs to read different numeric Ion types is not necessary in Python." ?

Reply:
Done.